rdklib/evaluation.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, complianceType, resourceId=None, resourceType=None, annotatio
         self.complianceType = complianceType
 
     def __repr__(self):
-        return f"Evaluation(annotation='{self.annotation}', complianceResourceId='{self.complianceResourceId}', complianceResourceType='{self.complianceResourceType}', complianceType='{self.complianceType}')"
+        return f"Evaluation(annotation='{self.annotation}', resourceId='{self.complianceResourceId}', resourceType='{self.complianceResourceType}', complianceType='{self.complianceType}')"
```

Comment:
Why are these lines changing? The Evaluation constructor is called with `complianceResourceId` and `complianceResourceType` as arguments. This change seems like it would add confusion.

Reply:
`complianceResourceId` and `complianceResourceType` are just class variables, not arguments.
The constructor (`__init__()`) is not called with `complianceResourceId` and `complianceResourceType`:
https://github.com/awslabs/aws-config-rdklib/blob/2247f5ea959fbe7a1243022b8e98e99262744605/rdklib/evaluation.py#L30

If we doesn't update that line, the test below will fail:
https://github.com/awslabs/aws-config-rdklib/pull/79/commits/acde2cd3eb48e5c5f7d9ca7437ed8b8ef0a4d1c4#diff-57f6d37c381e29e65e044de01f2888caf6707880f9c764eef63d4cd5e69749d8R191

I thought this fix is better than the alternative solution that fixes argument name of `__init__()` since it effects existing users.

Reply:
Yep, my bad -- was reading too fast. The change looks good. I'll get it added to the next release.